ivy/functional/frontends/paddle/nn/functional/loss.py

```diff
@@ -230,3 +230,21 @@ def margin_ranking_loss(input, other, label, margin=0.0, reduction="mean", name=
     out = ivy.atleast_1d(out)
 
     return out
+
+@with_supported_dtypes({"2.5.1 and below": ("float32","float64")},"paddle")
+@to_ivy_arrays_and_back
+def multilabel_soft_margin_loss(input, label, weight=None, reduction='mean', name=None):
```

Comment:
Function name is written incorrectly would you rename it to `multi_label_soft_margin_loss`